securedrop_client/storage.py

```diff
@@ -703,7 +708,7 @@ def update_draft_replies(
             and_(
                 DraftReply.source_id == source_id,
                 DraftReply.timestamp > timestamp,
-                DraftReply.file_counter == old_file_counter,
+                DraftReply.file_counter <= old_file_counter,
```

Comment:
Apply the `update_draft_replies` file_counter bump for _all_ drafts that have a newer timestamp and a smaller (or equal) file counter than the target.